tests/Support/SupportArrTest.php

```diff
@@ -99,8 +99,11 @@ public function testDivide()
 
     public function testDot()
     {
-        $array = Arr::dot(['foo' => ['bar' => 'baz']]);
-        $this->assertSame(['foo.bar' => 'baz'], $array);
+        $array = Arr::dot([10 => 100]);
```

Comment:
```suggestion
        $array = Arr::dot(['foo' => ['bar' => 'baz']]);
        $this->assertSame(['foo.bar' => 'baz'], $array);

        $array = Arr::dot([10 => 100]);
```

You shouldn't remove existing tests on a bugfix PR.